### DISCUSSION TITLE
Migration without internet connection

### DISCUSSION THREAD

**Discussion by markohrastovec** [oracle/centos2ol#140] [Q&A]
Can anybody give me some pointers how to migrate, if my server does not have access to internet? centos2ol script is by far the simplest way to migrate, but there is this limitation of internet connection. Is there a similar method to migrate offline or is my only way to reinstall?

**Comment by scoter-oracle** [comment]
Leveraging a proxy for your system could get you internet-access to your machine.
An other option could be to create your local yum-mirror for Oracle Linux; example available at https://docs.oracle.com/en/learn/local_yum-mirror_linux_8/
